tutorials/networking/high_level_multiplayer.rst

```diff
@@ -371,9 +372,151 @@ have loaded the game scene.
         players.clear()
         server_disconnected.emit()
 
+ .. code-tab:: csharp C#
+    using Godot;
+    using Godot.Collections;
+
+    public partial class Lobby : Node
```

Comment:
The GDScript example has a comment about this class being an autoload, that's important because otherwise the node won't be available in the tree (`/root/Lobby`). Maybe this should also be mentioned in the text and link to the [_Singletons (Autoload)_](https://docs.godotengine.org/en/stable/tutorials/scripting/singletons_autoload.html) documentation page.